client/src/components/Setter.jsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import axios from 'axios';
+import { connect } from 'react-redux';
+
+
+//A component to simulate arbitrary user clicks on products
+
+function Setter({
+  setProductInfo, setSelectedStyle, setMetaData, setStyles
+}) {
+  
+  function setDefaultStyle(styles) {
+    //console.log(styles);
+    const result = styles.filter((style) => style['default?'] === true);
+    return result;
+  }
+    
+  function handleClick() {
+    axios.get('/products/40348')
```

Comment:
It looks like the product to switch to is hardcoded here. Without a way to change this from the client this component doesn't serve much purpose.

Reply:
It makes more sense to me to keep the functionality of this component in `componentDidMount` inside of `Overview`.
Without any additional functionality I don't see a reason to separate it into its own component

Reply:
Once you guys build out the button/pictures that link to other comopnents, we cant just use a template literal in place of the hardcoded ID so that the function is called with the specific ID

Reply:
Or have it intialize data in App, that way it occurs when it mounts, but its still out of the client view